File: 2015/src/test/java/Day3Test.java
import nl.sbmf21.aoc15.Aoc;
import nl.sbmf21.aoc15.days.Day3;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class Day3Test {

    private final Day3 day = new Day3(new Aoc());

    @Test
    public void testPart1Output() {
        assertEquals(2081, (int) day.part1());
    }

    @Test
    public void testPart2Output() {
        assertEquals(2341, (int) day.part2());
    }
}

File: 2022/kotlin/src/test/kotlin/nl/sbmf21/aoc22/days/Day3Test.kt
package nl.sbmf21.aoc22.days

import nl.sbmf21.aoc22.testDay
import org.junit.jupiter.api.Test

class Day3Test {

    @Test
    fun testInput() = testDay(Day3::class.java, 7727, 2609)

    @Test
    fun testExample() = testDay(Day3::class.java, 157, 70, true)
}